### DISCUSSION TITLE
Deprecated endpoint

### DISCUSSION THREAD

**Discussion by drewvolz** [frog-pond/ccc-server#564] [Announcements]
> This news source is no longer being updated.

You're seeing this message because the url in this application is no longer being used. You may need to update the app on your device in order to avoid seeing this message.

**Comment by hawkrives** [comment]
Some data in All About Olaf is temporarily unavailable: currently the St. Olaf calendar, the A–Z index, and St. Olaf news tab. (The Oleville news tab is offline because Oleville is offline.)

St. Olaf’s website changed how it responds to us, so those three can’t be fetched the way the app currently does. 

We’re working on it, and there’s nothing to do on your end for now. This page will be updated when there is.
